drivers/usb/device/usb_dc_stm32.c

```diff
@@ -382,6 +374,15 @@ static int usb_dc_stm32_init(void)
 	usb_dc_stm32_state.pcd.Init.Sof_enable = 1;
 #endif /* CONFIG_USB_DEVICE_SOF */
 
+	LOG_DBG("Pinctrl signals configuration");
+	status = stm32_dt_pinctrl_configure(usb_pinctrl,
+				     ARRAY_SIZE(usb_pinctrl),
+				     (uint32_t)usb_dc_stm32_state.pcd.Instance);
+	if (status < 0) {
+		LOG_ERR("USB pinctrl setup failed (%d)", status);
+	return -EIO;
```

Comment:
better propagate error code

```suggestion
	return status;
```

Reply:
Fixed, txs